clang/setup.sh

```diff
@@ -35,6 +35,8 @@ usage () {
     echo "    -c,--only-check-install    check if recompiling clang is needed"
     echo "    -h,--help                  show this message"
     echo "    -r,--only-record-install   do not install clang but pretend we did"
+    echo "    -n|--ninja                 use ninja for building"
+    echo "    -s|--sequential-link       only use one process for linking (ninja only)"
```

Comment:
please follow the same still as the other options: `-n,--ninja` and `-s,--sequential-link`.

Reply:
Oops! That's definitely a `yyp` kind of error :p

Reply:
Also we should probably keep these sorted alphabetically.

Reply:
Both done!